### PR TITLE
Issue#155: Added checkstyle plugin.

### PR DESCRIPTION
**Pull request by VadymSokorenko**
-Pushing code with violations to test CI.

dev
## ZenHub

* [Main ZenHub ticket](https://app.zenhub.com/workspaces/home-project-5f7b77ff8db73a001cb17009)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: .github/checkstyle-configuration.xml [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+            <property name="tokens" value="ARRAY_DECLARATOR"/>
+            <property name="option" value="EOL"/>
+        </module>
+        <module name="SeparatorWrap">
+            <property name="id" value="SeparatorWrapMethodRef"/>
+            <property name="tokens" value="METHOD_REF"/>
+            <property name="option" value="nl"/>
+        </module>
+    </module>
+</module>
```
Comment (kh0ma) [inline .github/checkstyle-configuration.xml]: Please pay attention to leave an empty line at the end of the file. BTW can we require it from checkstyle perspective?

Comment (kh0ma) [reply .github/checkstyle-configuration.xml]: https://stackoverflow.com/a/5813359

**Review by kh0ma — APPROVED** [review]

File: .github/checkstyle-configuration.xml (hunk 1 above)
Comment (VadymSokorenko) [reply .github/checkstyle-configuration.xml]: Yes, we can. It's done by NewlineAtEndOfFile module with default properties.

**Review by alexorlenko — APPROVED** [review]

File: .github/checkstyle-configuration.xml (hunk 1 above)
Comment (Abhai2016) [reply .github/checkstyle-configuration.xml]: It works by default? Because I can't see the module with this name in the file above.

Comment (kh0ma) [reply .github/checkstyle-configuration.xml]: What is about this file? Just add a newline at the end

**Review by Abhai2016 — APPROVED** [review]

**Review by denis-litvinov — APPROVED** [review]

File: .github/checkstyle-configuration.xml (hunk 1 above)
Comment (VadymSokorenko) [reply .github/checkstyle-configuration.xml]: NewlineAtEndOfFile is not included in our configuration yet. I'll add it and a new line at the end of configuration file.

**Review by annaprok — APPROVED** [review]

**Pull request merged**